docroot/themes/custom/uids_base/scss/content/node--type--article.scss

```diff
@@ -149,6 +149,44 @@ article.node--type-article {
       margin-right: 0.3rem;
     }
   }
+
+  .block-field-blocknodearticlefield-image-caption {
+    font-style: italic;
+    color: #fff;
+    z-index: 3;
+    position: relative;
+    background: $dark-gray;
+    padding: 10px 20px;
+    order: 1;
```

Comment:
I'm not 100% about this, but I think it will infer the original order, so I don't think you don't need this.